soc/arm/nxp_imx/rt/soc_rt11xx.c

```diff
@@ -520,6 +526,63 @@ void imxrt_audio_codec_pll_init(uint32_t clock_name, uint32_t clk_src,
 }
 #endif
 
+#if CONFIG_MIPI_DSI
+void imxrt_pre_init_display_interface(void)
+{
+	/* elcdif output to MIPI DSI */
+	CLOCK_EnableClock(kCLOCK_Video_Mux);
+	VIDEO_MUX->VID_MUX_CTRL.CLR = VIDEO_MUX_VID_MUX_CTRL_MIPI_DSI_SEL_MASK;
```

Comment:
Here about the VIDEO_MUX, do we need to place it in another commit with a dts  defined?

Reply:
If there is a need for a video mux driver. At this point we do not need one.